src/list.cpp

```diff
@@ -53,6 +53,38 @@ bool IsCreateStatement(const std::string& sql_statement){
 
 // LOGICAL DATABASE DESIGN
 
+void CheckJSONValuedAttribute(Configuration& state,
+                              const std::string& sql_statement,
+                              bool& print_statement){
+
+  auto ddl_statement = IsDDLStatement(sql_statement);
+  if(ddl_statement == false){
+    return;
+  }
+  std::regex pattern("(\\s+hstore\\s+)|(\\s+json\\s+)");
```

Comment:
Since this is looking for `json` surrounded by whitespace, won't it ignore the following?

- the much more common `jsonb` type
- delcarations such as

  ```sql
  CREATE TABLE foo (
      id integer PRIMARY KEY,
      val json,
      key text
  );
  ```

  where `json` is followed by a comma

Not sure if there's some tokenization happening before, but even just `json` at the end of the line wouldn't match because it's not followed by a blank.